loadfile/loadfile.go

```diff
@@ -26,18 +28,17 @@ type fileCache struct {
 type FileCache interface {
 	RootDir() string
 	LoadContext() error
-	GetByKey(fileKey string) (*ContextFile, bool)
-	AbsPathByKey(fileKey string) *string
-	ContentByKey(fileKey string) []byte
+	GetByKey(fileKey string) (*ContextFile, error)
+	AbsPathByKey(fileKey string) (string, error)
+	ContentByKey(fileKey string) ([]byte, error)
 	Contents() map[string][]byte
-	AbsPaths() map[string]string
 	Files() map[string]ContextFile
 }
 
-// NewFileCacheUsingContext creates a mapping of files to their absolute paths and
-// file contents. rootDir is the context directory (root directory) in
-// which the files can be found. 'files' is a mapping of the desired file
-// key to a relative or absolute file path.
+// NewFileCacheUsingContext returns a FileCache object containing the provided root directory string and a
+// new map based on the provided map. The new map uses the same keys as the input but their corresponding
+// values contain absolute file paths. Relative file paths provided in the input map are prefixed with the root
+// directory before being included in the output map.
```

Comment:
If you make a change for some other reason...it would be nice to have this wrapped to 80 or so columns (like what you did for `NewFileCache()`.

Reply:
Won't do now. `gofmt` cannot be configured to do it. I do agree that it would be nice to have the line length standardized. It would be good to add this style formatting with [lll](https://golangci-lint.run/usage/linters/#lll) to our golangci configuration file.

Reply:
added as a new issue

Reply:
For what it's worth, line length is a rich topic without a pat answer.  Yes, you can configure a linter to try to enforce some sort of policy, but you can also configure your IDE to help or just to it manually.

Nevertheless, long lines in block comments are hard to read, particularly in code review, and there really isn't any need for them.  And, I noted that the new header comment for `NewFileCache()` was reasonably formatted, so it seemed like this one could be, as well.